spec/controllers/spree/api/products_controller_spec.rb

```diff
@@ -10,7 +10,7 @@ module Spree
     let(:product_other_supplier) { create(:product, supplier: supplier2) }
     let(:attributes) { [:id, :name, :supplier, :price, :on_hand, :available_on, :permalink_live] }
 
-    let(:current_api_user) { build_stubbed(:user) }
+    let(:current_api_user) { create(:user) }
```

Comment:
ahaha, nice. I tried build and it also works. build is faster than create. the spec runs in 9secs with build and 12s with create.